### PR TITLE
Rollback JFR plugin update

### PR DESCRIPTION
**Pull request by rinckm**
### Description

rollback jenkins update

### Submitter checklist

- [ ] Change has been tested (on a back-end cluster)
- [ ] (If applicable) Jira backlog item ID added to the PR title and the [changelog.yaml] entry
- [ ] [changelog.yaml] entry with upgrade notes is prepared and appropriate for the audience affected by the change (users or developer, depending on the change)
- [ ] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
    - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [ ] Links to external changelogs, since the last release of our component, added to the [changelog.yaml] entry (description).
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] Check if dependency updates affect our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There is at least 1 approval for the pull request and no outstanding requests for change
- [ ] All voter checks have passed
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] The Pull Request title is understandable and reflects the changes well
- [ ] The Pull Request description is understandable and well documented
- [ ] [changelog.yaml] entry for this Pull Request has been added
    - [ ] Changelog entry contains all required information
    - [ ] 'Upgrade notes' are documented in changelog.yaml (if required)

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

File: changelog.yaml [hunk 1]
```diff
@@ -51,7 +51,12 @@
 - version: NEXT
   date: TBD
   changes:
-
+    - type: enhancement
```
Comment (rinckm) [inline changelog.yaml:54]:
```suggestion
    - type: bug
```

File: changelog.yaml [hunk 2]
```diff
@@ -51,7 +51,12 @@
 - version: NEXT
   date: TBD
   changes:
-
+    - type: bug
+      impact: patch
+      title: Rollback updated plugins
+      description: |-
+        Rollback plugin update from 0.25.0
+      pullRequestNumber: 0
```
Comment (rinckm) [inline changelog.yaml:59]:
```suggestion
      pullRequestNumber: 358
```

File: changelog.yaml [hunk 3]
```diff
@@ -51,7 +51,12 @@
 - version: NEXT
   date: TBD
   changes:
-
+    - type: bug
+      impact: patch
+      title: Rollback updated plugins
+      description: |-
+        Rollback plugin update from 0.25.0
```
Comment (parsavali2) [inline changelog.yaml:58]:
```suggestion
        Rollback `stewardci/stewardci-jenkinsfile-runner` image to `230126_b29a3f0` version
```

File: changelog.yaml [hunk 4]
```diff
@@ -51,7 +51,12 @@
...
+      pullRequestNumber: 358
 - version: "0.25.0"
   date: 2023-01-26
```
Comment (parsavali2) [inline changelog.yaml:61]: You need to add a recall message to here if this version contains a bug

**Review by parsavali2 — CHANGES_REQUESTED** [review]

**Review by valiparsa — APPROVED** [review]

**Pull request merged**